telosnetwork/telos-wallet
# #326 | Display NFT Transfers

# Fixes #326 

## Description
This PR adds NFT transfers into the transfers tab

## Test scenarios
- this PR needs to be tested locally because we need to spoof the user account
- run `git fetch --all && git checkout 326-display-nft-xfers`
- change `.env` to `NETWORK="mainnet"`
- change the following lines in files:
    - `history.ts:264` - `const userAddressLower = '0x13B745FC35b0BAC9bab9fD20B7C9f46668232607'.toLowerCase();`
    - `nfts.ts:135` - `const owner = '0x13B745FC35b0BAC9bab9fD20B7C9f46668232607';`
    - `EVMChainSettings.ts:381` - `const address = '0x13B745FC35b0BAC9bab9fD20B7C9f46668232607';`
    - `EVMChainSettings.ts:463` - `const url = 'v1/account/0x13B745FC35b0BAC9bab9fD20B7C9f46668232607/transfers';`
- run `yarn dev`
- go to http://localhost:8081
- sign in
- go to the transactions tab
- navigate transactions and see NFTs displayed (easiest way to see this is to do 100 rows per page and go forward a few pages)
- click an NFT
    - you should be sent to NFT detail page


Also, see this video demonstrating the hover effect working at different positions on the screen:
https://github.com/telosnetwork/telos-wallet/assets/11837820/d53370dc-7e67-4181-9194-d87256976544



## Checklist:
